models/db.py

```diff
@@ -1,9 +1,36 @@
-from gluon.tools import Auth
 db = DAL('sqlite://storage.sqlite',pool_size=1,check_reserved=['all'])
```

Comment:
Please follow PEP8, there should be spaces between the arguments of a functions https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Pycharm highlights with yellow when something doesn't respect that, you can go on the respective and get tips regarding what is wrong.